### PR TITLE
Michael/update default file match pattern

### PR DESCRIPTION
**Pull request by mikicho**
1. Change the default match pattern to: `**/?(*.)+(spec|test).[jt]s` (copy what looked relevant from `jest`)
2. First argument is set as a file filter.

### CODE REVIEW

File: packages/testwatch/index.js [hunk 1]
```diff
@@ -220,7 +220,7 @@ ${Object.entries(this.#currentCommands)
   }
 
   async run() {
-    await this.#runTests();
+    this.#runTests();
```
Comment (MoLow) [inline packages/testwatch/index.js:223]: why?

Comment (mikicho) [reply packages/testwatch/index.js:223]: If we await the `runTests` function, we don't catch the `drained` event we fire where no file is found.

File: packages/testwatch/index.js [hunk 2]
```diff
@@ -220,7 +220,7 @@ ${Object.entries(this.#currentCommands)
...
     await once(this.#emitter, 'drained');
```
Comment (MoLow) [inline packages/testwatch/index.js:224]:
```suggestion
    await Promise.race([this.#runTests(), once(this.#emitter, 'drained')]
```
I think this should be the solution - but it should probably be done in a separate PR?

Comment (mikicho) [reply packages/testwatch/index.js:224]: Great solution!
Why separate?

Comment (mikicho) [reply packages/testwatch/index.js]: ok. I pushed a revert for this change and will open another PR

**Review by MoLow — APPROVED** [review]
commits need to be squashed and amended to adhere conventional commit

**Comment by mikicho** [comment]
> commits need to be squashed and amended to adhere conventional commit

@MoLow, How do you "squash all commits in branch"? for now, I used `rebase -i`, but it was cumbersome.

**Comment by MoLow** [comment]
`git rebase -i` works, or `git comit reset --soft main & git commit`
